pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancer.java

```diff
@@ -1145,8 +1174,8 @@ static boolean isExternalViewConverged(String tableNameWithType,
         }
       }
 
-      // For low disk mode, check if there are extra instances in ExternalView that are not in IdealState
-      if (lowDiskMode && externalViewInstanceStateMap != null) {
+      // Check if there are extra instances in ExternalView that are not in IdealState
+      if (externalViewInstanceStateMap != null) {
```

Comment:
maybe open a separate small PR for this fix for some closer reviews (and perhaps also include those simple typo fixes like segmentKey -> instanceName)

Reply:
done - removed this. will open a separate PR later